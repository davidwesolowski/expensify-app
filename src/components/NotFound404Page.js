import React from 'react';
import { Link } from 'react-router-dom';

const NotFound404Page = () => (
    <div>
        <h3>Page not found 404</h3>
        <Link to="/">Go home</Link>
    </div>
);

export default NotFound404Page;