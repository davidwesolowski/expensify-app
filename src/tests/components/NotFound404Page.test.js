import React from 'react';
import { shallow } from 'enzyme';
import NotFound404Page from '../../components/NotFound404Page';

test('Should render NotFound404Page correctly', () =>
{
    const wrapper = shallow(<NotFound404Page />);
    expect(wrapper).toMatchSnapshot();
});