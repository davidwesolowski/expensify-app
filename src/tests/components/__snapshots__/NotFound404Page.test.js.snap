// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render NotFound404Page correctly 1`] = `
<div>
  <h3>
    Page not found 404
  </h3>
  <Link
    to="/"
  >
    Go home
  </Link>
</div>
`;
